### ISSUE TITLE
Unable to convert any of the YOLO tf models

### ISSUE THREAD

**Issue by antoniomtz** [openvinotoolkit/open_model_zoo#2816]
Hello,

I'm trying to convert the YOLO models. After I download them, I run:

`./converter.py --name yolo-v3-tf`

And get the following error:

> ========== Converting yolo-v3-tf to IR (FP16)
> Conversion command: /home/antonio/miniconda3/bin/python3 -m mo --framework=tf --data_type=FP16 --output_dir=/home/antonio/open_model_zoo/tools/downloader/public/yolo-v3-tf/FP16 --model_name=yolo-v3-tf '--input_shape=[1,416,416,3]' --input=input_1 '--scale_values=input_1[255]' --reverse_input_channels --transformations_config=/home/antonio/open_model_zoo/tools/downloader/public/yolo-v3-tf/yolo-v3.json --input_model=/home/antonio/open_model_zoo/tools/downloader/public/yolo-v3-tf/yolo-v3.pb
> 
> /home/antonio/miniconda3/bin/python3: No module named mo.__main__; 'mo' is a package and cannot be directly executed
> FAILED:
> yolo-v3-tf

I'm using Python 3.8.5 and this is the pip list:

> Package                 Version             Location
> ----------------------- ------------------- -------------------------------------
> absl-py                 0.14.1
> addict                  2.4.0
> astroid                 2.5.6
> astunparse              1.6.3
> attrs                   20.3.0
> autopep8                1.5.7
> brotlipy                0.7.0
> cachetools              4.2.4
> certifi                 2020.6.20
> cffi                    1.14.3
> chardet                 3.0.4
> charset-normalizer      2.0.6
> conda                   4.9.2
> conda-package-handling  1.7.2
> cryptography            3.2.1
> flatbuffers             1.12
> gast                    0.3.3
> google-auth             1.35.0
> google-auth-oauthlib    0.4.6
> google-pasta            0.2.0
> grpcio                  1.32.0
> h5py                    2.10.0
> idna                    2.10
> iniconfig               1.1.1
> isort                   5.8.0
> joblib                  1.0.1
> jsonschema              3.2.0
> Keras-Preprocessing     1.1.2
> lazy-object-proxy       1.6.0
> Markdown                3.3.4
> mccabe                  0.6.1
> mmcv                    1.3.4
> numpy                   1.19.5
> oauthlib                3.1.1
> opt-einsum              3.3.0
> ote                     0.2                 /home/antonio/training_extensions/ote
> packaging               20.9
> Pillow                  8.2.0
> pip                     20.2.4
> pluggy                  0.13.1
> protobuf                3.18.1
> py                      1.10.0
> pyasn1                  0.4.8
> pyasn1-modules          0.2.8
> pycodestyle             2.7.0
> pycosat                 0.6.3
> pycparser               2.20
> pylint                  2.7.2
> pyOpenSSL               19.1.0
> pyparsing               2.4.7
> pyrsistent              0.17.3
> PySocks                 1.7.1
> pytest                  6.2.4
> PyYAML                  5.4.1
> requests                2.26.0
> requests-oauthlib       1.3.0
> rsa                     4.7.2
> ruamel-yaml             0.15.87
> scikit-learn            0.23.1
> scipy                   1.6.2
> setuptools              50.3.1.post20201107
> Shapely                 1.7.0
> six                     1.15.0
> tensorboard             2.6.0
> tensorboard-data-server 0.6.1
> tensorboard-plugin-wit  1.8.0
> tensorflow              2.4.3
> tensorflow-estimator    2.4.0
> termcolor               1.1.0
> texttable               1.6.3
> threadpoolctl           2.1.0
> toml                    0.10.2
> tqdm                    4.51.0
> typing-extensions       3.7.4.3
> urllib3                 1.25.11
> Werkzeug                2.0.2
> wheel                   0.35.1
> wrapt                   1.12.1
> yapf                    0.31.0

Any suggestions? Thanks.

**Comment by vladimir-dudnik** [comment]
@antoniomtz you probably work with Open Model Zoo from `develop` branch, I'd recommend you either use Open Model Zoo from OpenVINO install package or clone Open Model Zoo `master` branch by tag for corresponding OpenVINO release. For example, for the latest OpenVINO 2021.1 release use tag `2021.4.1`

**Comment by antoniomtz** [comment]
@vladimir-dudnik You are right. I matched the OMZ with my openVINO version and I was able to convert the model and use it nicely with DLstreamer. Thank you
